src/amo/components/MastHead.js

```diff
@@ -35,15 +30,12 @@ export class MastHeadBase extends React.Component {
         </div>
         {children}
         <header className="MastHead-header">
-          {isHomePage
-            ? <h1 ref={(ref) => { this.title = ref; }}
-                className="MastHead-title MastHead-homepage">
-              {headerTitle}
-            </h1>
-            : <Link ref={(ref) => { this.title = ref; }} to="/"
-                className="MastHead-title">
-              {headerTitle}
-            </Link>}
+          <h1>
```

Comment:
Gotcha. I guess the `<h1>` on the homepage makes sense. I like the discoverability realising "oh this is a link it will always get me home" by it being a link on the homepage. If a link to home on home is considered _bad_ UX I can certainly remove it, but I find it handy.